storage/src/tests/visiting/visitormanagertest.cpp

```diff
@@ -78,8 +82,9 @@ class VisitorManagerTest : public CppUnit::TestFixture
             std::vector<document::Document::SP >& docs,
             std::vector<document::DocumentId>& docIds,
             api::ReturnCode::Result returnCode = api::ReturnCode::OK,
-            documentapi::Priority::Value priority = documentapi::Priority::PRI_NORMAL_4);
+            std::optional<Priority::Value> priority = documentapi::Priority::PRI_NORMAL_4);
```

Comment:
Optional with a default value ?

Reply:
A bit of a hack, I'll admit 👀 This value is used for an assertion that we want to check in 99% of the cases, but some times not. We send in an empty optional in that 1%.